app.json

```diff
@@ -33,6 +33,24 @@
     },
     "SMTP_USERNAME":{
```

Comment:
Do we need these other mail-related ENV variables since Sendgrid is installed now?